plugins/start-alpine.yaml

```diff
@@ -0,0 +1,13 @@
+  start-alpine-pod:
+    shortCut: Ctrl-T
+    confirm: true
+    description: "Start an alpine:latest pod in current context/namespace"
+    scopes:
+      - pods
+      - deployments
+    command: bash
+    background: true
+    args:
+      - -c
+      - |
+        echo '{"apiVersion": "apps/v1", "kind": "Deployment", "metadata": {"name": "alpine", "labels": {"app": "alpine", "debug": "1"}}, "spec": {"selector": {"matchLabels": {"app": "alpine"}}, "replicas": 1, "template": {"metadata": {"labels": {"app": "alpine", "debug": "1", "sidecar.istio.io/inject": "false"}}, "spec": {"containers": [{"name": "alpine", "image": "alpine:latest", "imagePullPolicy": "Always", "securityContext": {"runAsUser": 0, "runAsGroup": 0}, "stdin": true, "tty": true, "stdinOnce": true, "terminationMessagePath": "/dev/termination-log", "terminationMessagePolicy": "File", "resources": {"requests": {"cpu": "100m", "memory": "100Mi"}, "limits": {"cpu": "100m", "memory": "100Mi"}}}], "restartPolicy": "Always"}}}}' | kubectl apply -f - --context $CONTEXT --namespace $NAMESPACE
```

Comment:
@sbordeyne Why do this assumes the istio label?

Reply:
Lots of people run istio, but having the prompt to select a container because of the sidecar is inconvenient.

If k9s follows the `kubectl.kubernetes.io/default-container` annotation it could be another way to address the issue. I made this plugin for myself (on a setup that runs istio), and I do want to skip sidecar injection for tests (I want a pod that starts fast, that I can shell into instantly)

Reply:
I think k9s now checks the default container annotation when you shell into a pod with x containers see (#3224)

I get the use case but might good to mention in the plugin docs the istio use case since not only this is a deployment and not a single pod flash/debug container, it could potentially introduce a security hole in the target cluster since it's not injected and hence not guarded by any istio policies....

Reply:
Nice, since that's the case I'll remove the istio annotation since having a quick shell in a pod for tests is what this plugin is for.

As for the single pod vs deployment, it's done this way so that people can edit the deployment to add in volume mounts if need be (if you need to debug PVCs) or to add in a service account to the pod (to troubleshoot RBAC issues)